nservicebus/rabbitmq/connection-settings_tls-details_rabbit_[4,].partial.md

```diff
@@ -0,0 +1,13 @@
+## Transport Layer Security support
+
+Secure connections to the broker using Transport Layer Security (TLS) are supported. For information on how to configure TLS on the broker, refer to the [RabbitMQ documentation](http://www.rabbitmq.com/ssl.html). To enable TLS support, set the `UseTls` setting to `true` in the connection string. If the broker has been configured to require client authentication, a client certificate can be specified in the `CertPath` setting. If that certificate requires a password, it can be specified in the `CertPassphrase` setting.
```

Comment:
I think make the text "transport layer security (tls)" link to the rabbit doco and drop the "for information" sentence